### ISSUE TITLE
Asking for a pre-trained model

### ISSUE THREAD

**Issue by MondayOFF22** [SSinyu/WGAN_VGG#14]
Hello,sir.I successfully run the code, but the loss function I get from training is negative. And the prediction results are worse than LDCT. could you share your pre-trained model parameters with a .ckpt file? Thanks a lot!My email is: ngp20001010@163.com
